src/VkBootstrap.h

```diff
@@ -115,6 +115,34 @@ template <typename T> class Result {
 	bool m_init;
 };
 
+struct GenericFeaturesPNextNode {
+
+    GenericFeaturesPNextNode() : sType(static_cast<VkStructureType>(0)),
+                                 pNext(nullptr) {
+        memset(fields, 0, sizeof(fields));
+    }
+
+    VkStructureType sType = static_cast<VkStructureType>(0);
+    void* pNext = nullptr;
+    VkBool32 fields[256];
```

Comment:
Would be good to create a static uint32_t member variable to store the number of fields.
```c
static const uint32_t USE_BETTER_NAME = 256;
VkBool32 fields[USE_BETTER_NAME];
```